### ISSUE TITLE
Move net.sql, net.pg, net,mysql to rocks

### ISSUE THREAD

**Issue by rtsisyk** [tarantool/tarantool#324]
```
tarantool> require('box.net.sql')
---
- true <!-- WTF?
...
```

Why do we need extra wrapper for pg/mysl modules?
It it possible to convert box.net.sql to the following structure:

```
pg  = require('pg')
conn = pg.connect(params)
conn.select(...)
```

and 

```
mysql  = require('mysql')
conn = pg.connect(params)
conn.select(...)
```

instead of 
```
conn = box.net.sql.connect('pg', args)
conn.select('xxx')
```

I propose to hide this super-module from the user.

**Comment by rtsisyk** [comment]
I need stable coeio_custom() API to do that.

**Comment by rtsisyk** [comment]
Fixed by 732c467efd5edfce70480ae3ae4d17cd180dc90b
